packages/ag-charts-community/src/util/destroy.ts

```diff
@@ -0,0 +1,14 @@
+type Destroyable = {} | { destroy(): void };
```

Comment:
Why allow objects without a `destroy()` - I think that will lead to more cases where a missing `destroy()` is not flagged up?

Reply:
ACK. Some of the objects in `ChartContext` do not have a destructors, but we can add empty `destroy()` methods to these.